### PR TITLE
View voter list as it is in voter file upload

### PR DESCRIPTION
**Pull request by edmarmartineli**
In my helios voting instance, I registered 170 users. The users registered 980 elections and
500,000 voters. It is easier for users to check the voters list if the fields are the same and in the same order as they appear in the voter file upload.

### CODE REVIEW

**Comment by edmarmartineli** [comment]
Please wait to evaluate this change. I need to test it more.

**Comment by edmarmartineli** [comment]
I've done more tests. You can analyze my improvement suggestion now. Thank you.

File: helios/templates/voters_list.html [hunk 1]
```diff
@@ -123,17 +128,20 @@ <h3 class="title">{{election.name}} &mdash; Voters and Ballot Tracking Center <s
...
 {% if admin_p %}
+<td style="white-space: nowrap;">
 [<a href="{% url "helios.views.voters_email" election.uuid %}?voter_id={{voter.voter_login_id}}">email</a>]
 [<a onclick="return confirm('are you sure you want to remove {{voter.name}} ?');" href="{% url "helios.views.voter_delete" election.uuid voter.uuid %}">x</a>]
+</td>
+<td>{{voter.voter_login_id}}</td>
+<td>{{voter.voter_email}}</td>
 {% endif %}
-<img class="small-logo" src="/static/auth/login-icons/{{voter.voter_type}}.png" alt="{{voter.voter_type}}" /> {{voter.name}}</td>
+<td>{{voter.name}}</td>
```
Comment (benadida) [inline helios/templates/voters_list.html]: why remove the user type logo?

File: helios/templates/voters_list.html [hunk 2]
```diff
@@ -123,17 +128,20 @@ <h3 class="title">{{election.name}} &mdash; Voters and Ballot Tracking Center <s
...
 {% endif %}
 {% if election.use_voter_aliases %}
 <td>{{voter.alias}}</td>
 {% endif %}
-<td><tt style="font-size: 1.4em;;">{% if voter.vote_hash %}{{voter.vote_hash}} <span style="font-size:0.8em;">[<a href="{% url "helios.views.castvote_shortcut" vote_tinyhash=voter.vote_tinyhash %}">view</a>]</span>{% else %}&mdash;{% endif %}</tt></td>
+<td>{% if voter.vote_hash %}{{voter.vote_hash}} <span style="font-size:0.8em;">[<a href="{% url "helios.views.castvote_shortcut" vote_tinyhash=voter.vote_tinyhash %}">view</a>]</span>{% else %}&mdash;{% endif %}</td>
```
Comment (benadida) [inline helios/templates/voters_list.html]: why remove the fixed-width `tt` ?

File: helios/templates/voters_list.html (hunk 1 above)
Comment (edmarmartineli) [reply helios/templates/voters_list.html]: I do not think this is important information, but it is. I'll undo this change.

File: helios/templates/voters_list.html (hunk 2 above)
Comment (edmarmartineli) [reply helios/templates/voters_list.html]: To leave the formatted text equal to the other fields. The tt tag is also deprecated and its use discouraged.

Comment (benadida) [reply helios/templates/voters_list.html]: I'd be okay with moving away from `tt`, but can we keep it fixed-width font?

Comment (edmarmartineli) [reply helios/templates/voters_list.html]: OK no problems. I'll leave the code as it was. But why is it necessary to keep it fixed-width tont?

**Pull request merged**